LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ModuleConnectionHost.cs

```diff
@@ -156,7 +161,8 @@ internal Task OnDesiredPropertiesUpdate(TwinCollection desiredProperties, object
             {
                 this.logger.LogWarning($"A desired properties update was detected but the parameters are out of range with exception :  {ex}");
             }
-            catch (Exception ex) when (ExceptionFilterUtility.False(() => this.logger.LogError($"An exception occurred on desired property update: {ex}")))
+            catch (Exception ex) when (ExceptionFilterUtility.False(() => this.logger.LogError(ex, $"An exception occurred on desired property update: {ex}"),
+                                                                    () => this.processingErrorCounter?.Add(1)))
```

Comment:
Is the safe navigation still required? Because I see in the tests you pass a `TestMeter` instance

Reply:
Removed with 567bfc030f0a954c1170638a56e74b26e6c2f964